services/forwarding/forwarding.rst

```diff
@@ -52,8 +52,8 @@ icon next to the Catch-All address.
 Catch-All Example Use Case
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Traditionally individual mailboxes or distribution lists needed to be configured
-for each address in a domain. If a small business wanted to add a new address,
+Traditionally individual mailboxes need to be configured
```

Comment:
for the future - this use case is for hosting email, but mailsac is a QA testing service